deploy/operator/monitoring-stack-operator-rules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/name: monitoring-stack-operator
+    app.kubernetes.io/part-of: monitoring-stack-operator
+    prometheus: k8s
+    role: alert-rules
+  name: monitoring-stack-operator-prometheus-rules
+spec:
+  groups:
+  - name: monitoring-stack-operator.rules
+    rules:
+    - alert: MonitoringStackOperatorReconcileErrors
+      annotations:
+        description: MonitoringStackOperator failing to reconcile controller {{ $labels.controller
+              }}
+        summary: MonitoringStack Operator failing to reconcile controller {{ $labels.controller
```

Comment:
Same - `Monitoring Stack Operator controller - {{$labels.controller}} fails to reconcile`